src/redis_cmd.cc

```diff
@@ -32,6 +32,8 @@
 
 namespace Redis {
 
+const std::string kCursorPrefix = "_";
```

Comment:
I think we should add comments for `scan` in doc/commands to say our cursor is not a muber, but 0 always is the end of cursor